src/components/LogoArtwork/styles.scss

```diff
@@ -0,0 +1,52 @@
+// Mobile Portrait
+.wrapper {
+	display: flex;
+	position: absolute;
+	height: 100%;
+	width: 100%;
```

Comment:
Good use of the aspect-ratio stuff below, but I think that the wrapper should have the height/width defined down there - and the children should just be constrained to its own size; otherwise the implication is that the Logo Artwork fills the screen, when in-fact it doesn't.   To center it, we can just do our usual `transform` trick...

Reply:
Cool done, thanks!